inst/examples/DT-searchExact/server.R

```diff
@@ -0,0 +1,56 @@
+library(shiny)
+library(DT)
+
+function(input, output, session) {
+
+  dat <- data.frame(
+    car = c("Mazda", "Mazda RX4", "Mazda RX4 Wag", "Ford", "Mercedes"),
+    pet = c("dog", "dog", "cat", "cat", "cat")
+  )
+
+  output[["dtable1"]] <- renderDT({
+
+    datatable(
+      dat, filter = "top",
+      options = list(
+        dom = "t",
+        columnDefs = list(
+          list(targets = "_all", className = "dt-center")
+        )
+      )
+    )
+
+  })
+
+  output[["dtable2"]] <- renderDT({
+
+    js <- c(
+      "function(settings) {",
+      "  var instance = settings.oInstance;",
+      "  var table = instance.api();",
+      "  var $inputs = instance.parent().find('.form-group input');",
+      "  $inputs.off('keyup search input').on('keyup', function() {",
+      "    var value = $(this).val();",
+      "    if(value !== '') {",
+      "      value = '^' + value + '$';",
+      "      var index = 1 + $inputs.index(this);", # add one for rownames column
+      "      var column = table.column(index);",
+      "      column.search(value, true, false).draw();",
+      "    }",
+      "  });",
+      "}"
```

Comment:
For large amounts of JS code, I wonder if it'd better to write them in a separate JS file and read the lines here. That might make the JS code easier to read and develop.

Reply:
But is it possible to make this JS file visible with the `showcase` option of Shiny?

Reply:
Good point. No.

Okay, I'll just merge it as is. Thanks!